mxnet-bot/LabelBotAddLabels/LabelBot.py

```diff
@@ -69,28 +76,31 @@ def count_pages(self, obj, state='all'):
         assert obj in set(["issues", "labels"]), "Invalid Input!"
         url = 'https://api.github.com/repos/{}/{}'.format(self.repo, obj)
         if obj == 'issues':
-            response = requests.get(url, {'state': state}, auth=self.auth)    
+            response = requests.get(url, {'state': state,
+                                          'per_page': 100}, auth=self.auth)
```

Comment:
As far as I know, even though you set `"per_page": 100`, but github will still display 30 issues at max per page.  You can print `len(response.json())` to double check.

Reply:
So please take a look at this example:
```python
    url = 'https://api.github.com/repos/{}/{}'.format(ts.repo, 'issues')
    response = requests.get(url,
                                    {'state': 'open',
                                     'base': 'master',
                                     'sort': 'created',
                                     'direction': 'desc',
                                     'page': 2,
                                     'per_page': 100
                                     # 'q': 'no:label+is:open'
                                     },
                                     auth=ts.auth)
    print(response.headers)
    print(len(response.json()))
    for ele in response.json():
        print("https://github.com/apache/incubator-mxnet/issues/" + ele['url'].split('/')[-1])
    res = requests.get("https://api.github.com/repos/apache/incubator-mxnet/labels", auth = ts.auth)
    print(res.json())
```

```
{'date': 'Tue, 14 Aug 2018 19:39:35 GMT', 'content-type': 'application/json; charset=utf-8', 'transfer-encoding': 'chunked', 'server': 'GitHub.com', 'status': '200 OK', 'x-ratelimit-limit': '5000', 'x-ratelimit-remaining': '4999', 'x-ratelimit-reset': '1534279175', 'cache-control': 'private, max-age=60, s-maxage=60', 'vary': 'Accept, Authorization, Cookie, X-GitHub-OTP, Accept-Encoding', 'etag': 'W/"ff6080df6b1349172187dc037379145f"', 'x-oauth-scopes': 'repo', 'x-accepted-oauth-scopes': 'repo', 'x-github-media-type': 'github.v3; format=json', 'link': '<https://api.github.com/repositories/34864402/issues?state=open&base=master&sort=created&direction=desc&page=1&per_page=100>; rel="prev", <https://api.github.com/repositories/34864402/issues?state=open&base=master&sort=created&direction=desc&page=3&per_page=100>; rel="next", <https://api.github.com/repositories/34864402/issues?state=open&base=master&sort=created&direction=desc&page=10&per_page=100>; rel="last", <https://api.github.com/repositories/34864402/issues?state=open&base=master&sort=created&direction=desc&page=1&per_page=100>; rel="first"', 'access-control-expose-headers': 'ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval', 'access-control-allow-origin': '*', 'strict-transport-security': 'max-age=31536000; includeSubdomains; preload', 'x-frame-options': 'deny', 'x-content-type-options': 'nosniff', 'x-xss-protection': '1; mode=block', 'referrer-policy': 'origin-when-cross-origin, strict-origin-when-cross-origin', 'content-security-policy': "default-src 'none'", 'x-runtime-rack': '1.437391', 'content-encoding': 'gzip', 'x-github-request-id': '3FC1:6DD4:2D0DD4:3B027C:5B732FF6'}
100
https://github.com/apache/incubator-mxnet/issues/11949
https://github.com/apache/incubator-mxnet/issues/11948
https://github.com/apache/incubator-mxnet/issues/11945
https://github.com/apache/incubator-mxnet/issues/11944
https://github.com/apache/incubator-mxnet/issues/11943
https://github.com/apache/incubator-mxnet/issues/11938
https://github.com/apache/incubator-mxnet/issues/11933
https://github.com/apache/incubator-mxnet/issues/11932
https://github.com/apache/incubator-mxnet/issues/11926
https://github.com/apache/incubator-mxnet/issues/11925
https://github.com/apache/incubator-mxnet/issues/11921
https://github.com/apache/incubator-mxnet/issues/11919
https://github.com/apache/incubator-mxnet/issues/11918
https://github.com/apache/incubator-mxnet/issues/11916
https://github.com/apache/incubator-mxnet/issues/11914
https://github.com/apache/incubator-mxnet/issues/11913
https://github.com/apache/incubator-mxnet/issues/11912
https://github.com/apache/incubator-mxnet/issues/11906
https://github.com/apache/incubator-mxnet/issues/11905
https://github.com/apache/incubator-mxnet/issues/11904
https://github.com/apache/incubator-mxnet/issues/11903
https://github.com/apache/incubator-mxnet/issues/11901
https://github.com/apache/incubator-mxnet/issues/11898
https://github.com/apache/incubator-mxnet/issues/11896
https://github.com/apache/incubator-mxnet/issues/11895
https://github.com/apache/incubator-mxnet/issues/11894
https://github.com/apache/incubator-mxnet/issues/11891
https://github.com/apache/incubator-mxnet/issues/11890
https://github.com/apache/incubator-mxnet/issues/11885
https://github.com/apache/incubator-mxnet/issues/11884
https://github.com/apache/incubator-mxnet/issues/11883
https://github.com/apache/incubator-mxnet/issues/11880
https://github.com/apache/incubator-mxnet/issues/11879
https://github.com/apache/incubator-mxnet/issues/11878
https://github.com/apache/incubator-mxnet/issues/11875
https://github.com/apache/incubator-mxnet/issues/11873
https://github.com/apache/incubator-mxnet/issues/11871
https://github.com/apache/incubator-mxnet/issues/11868
https://github.com/apache/incubator-mxnet/issues/11867
https://github.com/apache/incubator-mxnet/issues/11865
https://github.com/apache/incubator-mxnet/issues/11859
https://github.com/apache/incubator-mxnet/issues/11858
https://github.com/apache/incubator-mxnet/issues/11855
https://github.com/apache/incubator-mxnet/issues/11854
https://github.com/apache/incubator-mxnet/issues/11849
https://github.com/apache/incubator-mxnet/issues/11847
https://github.com/apache/incubator-mxnet/issues/11846
https://github.com/apache/incubator-mxnet/issues/11844
https://github.com/apache/incubator-mxnet/issues/11843
https://github.com/apache/incubator-mxnet/issues/11842
https://github.com/apache/incubator-mxnet/issues/11841
https://github.com/apache/incubator-mxnet/issues/11840
https://github.com/apache/incubator-mxnet/issues/11839
https://github.com/apache/incubator-mxnet/issues/11838
https://github.com/apache/incubator-mxnet/issues/11836
https://github.com/apache/incubator-mxnet/issues/11835
https://github.com/apache/incubator-mxnet/issues/11831
https://github.com/apache/incubator-mxnet/issues/11819
https://github.com/apache/incubator-mxnet/issues/11817
https://github.com/apache/incubator-mxnet/issues/11816
https://github.com/apache/incubator-mxnet/issues/11809
https://github.com/apache/incubator-mxnet/issues/11804
https://github.com/apache/incubator-mxnet/issues/11802
https://github.com/apache/incubator-mxnet/issues/11801
https://github.com/apache/incubator-mxnet/issues/11796
https://github.com/apache/incubator-mxnet/issues/11794
https://github.com/apache/incubator-mxnet/issues/11785
https://github.com/apache/incubator-mxnet/issues/11778
https://github.com/apache/incubator-mxnet/issues/11777
https://github.com/apache/incubator-mxnet/issues/11774
https://github.com/apache/incubator-mxnet/issues/11773
https://github.com/apache/incubator-mxnet/issues/11769
https://github.com/apache/incubator-mxnet/issues/11758
https://github.com/apache/incubator-mxnet/issues/11756
https://github.com/apache/incubator-mxnet/issues/11753
https://github.com/apache/incubator-mxnet/issues/11743
https://github.com/apache/incubator-mxnet/issues/11742
https://github.com/apache/incubator-mxnet/issues/11740
https://github.com/apache/incubator-mxnet/issues/11738
https://github.com/apache/incubator-mxnet/issues/11736
https://github.com/apache/incubator-mxnet/issues/11734
https://github.com/apache/incubator-mxnet/issues/11733
https://github.com/apache/incubator-mxnet/issues/11729
https://github.com/apache/incubator-mxnet/issues/11728
https://github.com/apache/incubator-mxnet/issues/11727
https://github.com/apache/incubator-mxnet/issues/11726
https://github.com/apache/incubator-mxnet/issues/11725
https://github.com/apache/incubator-mxnet/issues/11724
https://github.com/apache/incubator-mxnet/issues/11723
https://github.com/apache/incubator-mxnet/issues/11720
https://github.com/apache/incubator-mxnet/issues/11719
https://github.com/apache/incubator-mxnet/issues/11718
https://github.com/apache/incubator-mxnet/issues/11716
https://github.com/apache/incubator-mxnet/issues/11715
https://github.com/apache/incubator-mxnet/issues/11713
https://github.com/apache/incubator-mxnet/issues/11711
https://github.com/apache/incubator-mxnet/issues/11710
https://github.com/apache/incubator-mxnet/issues/11709
https://github.com/apache/incubator-mxnet/issues/11707
https://github.com/apache/incubator-mxnet/issues/11704
```

Reply:
Nice!